wireapp/wire-android
# fix: call muted after some seconds when joining (AR-3418)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-3418" title="AR-3418" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />AR-3418</a>  Joined a large conference call with mic open
  </td></table>
  <br />
 












----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

When joining a large conference call, the microphone stays unmuted for some seconds.

### Causes (Optional)

Muting calls was done after the call got established, and this could take some time (~4s) especially if the it's a large conference call

### Solutions

Part of this fix was done in this kalium [PR](https://github.com/wireapp/kalium/pull/1744)

In AR, we made preview calling screens do not mute device microphone 🎤 

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Try to join a large conference call.
Your mic should be muted immediately after your join


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
